### DISCUSSION TITLE
Text color in the report

### DISCUSSION THREAD

**Discussion by ratelChief** [btd/rollup-plugin-visualizer#184] [Q&A] [answered]
![image](https://github.com/btd/rollup-plugin-visualizer/assets/34477573/f8b7a3a1-db21-47b2-aaeb-8e195a5b5c83)
why some text is white and other is black? or it's just a visual effect for better readability?

**Comment by btd** [comment, marked as answer]
Yes, it is for better readability. See there https://github.com/btd/rollup-plugin-visualizer/blob/master/src/treemap/color.ts#L19C20-L19C20
It checks luminance of background and tries to select either white or black.
